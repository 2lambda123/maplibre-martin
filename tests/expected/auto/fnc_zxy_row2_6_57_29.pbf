E
MixedCase.function_Mixed_Name	"	œ¦7	"	œ¨7	"	š¦7(€ x