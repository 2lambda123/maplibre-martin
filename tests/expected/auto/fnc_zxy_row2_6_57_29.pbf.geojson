{
  "features": [
    {
      "geometry": {
        "coordinates": [
          1614,
          557
        ],
        "type": "Point"
      },
      "properties": {
        "source_mvt_layer": "MixedCase.function_Mixed_Name"
      },
      "type": "Feature"
    },
    {
      "geometry": {
        "coordinates": [
          1614,
          556
        ],
        "type": "Point"
      },
      "properties": {
        "source_mvt_layer": "MixedCase.function_Mixed_Name"
      },
      "type": "Feature"
    },
    {
      "geometry": {
        "coordinates": [
          1613,
          557
        ],
        "type": "Point"
      },
      "properties": {
        "source_mvt_layer": "MixedCase.function_Mixed_Name"
      },
      "type": "Feature"
    }
  ],
  "name": "merged",
  "type": "FeatureCollection"
}
