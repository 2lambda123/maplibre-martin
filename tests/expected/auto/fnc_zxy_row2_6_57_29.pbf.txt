=============================================================
layer: 0
  name: MixedCase.function_Mixed_Name
  version: 2
  extent: 4096
  feature: 0
    id: (none)
    geomtype: point
    geometry:
      POINT(1614,3539)
    properties:
  feature: 1
    id: (none)
    geomtype: point
    geometry:
      POINT(1614,3540)
    properties:
  feature: 2
    id: (none)
    geomtype: point
    geometry:
      POINT(1613,3539)
    properties:
